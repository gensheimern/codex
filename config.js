module.exports = {
	DB_HOST: process.env.DB_HOST || '159.65.115.221',
	DB_USER: process.env.DB_USER || 'root',
	DB_PASS: process.env.DB_PASS || 'teamcodex',
	DB_NAME: process.env.DB_NAME || 'lunch_planner',
	JWT_SECRET: process.env.JWT_SECRET || 'secret',
	MAIL_HOST: process.env.MAIL_HOST || '159.65.115.221',
	MAIL_USER: process.env.MAIL_USER || 'support@codex-team.de',
	MAIL_PASS: process.env.MAIL_PASS || 'teamcodex123',
};
