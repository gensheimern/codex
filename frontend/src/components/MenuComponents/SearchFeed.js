import React from 'react';
import Drawer from 'material-ui/Drawer';
import MenuItem from 'material-ui/MenuItem';
import FlatButton from 'material-ui/FlatButton';
import IconSearch from 'material-ui/svg-icons/action/search';
import "./sidebars.css"

const SearchIcon = <IconSearch/>;

export default class SearchFeed extends React.Component {

  constructor(props) {
    super(props);
    this.state = {open: false};
  }

  handleClick = () => console.log("serached for nothing!");



  render() {
    return (
      <div className="SearchFeed">
        <FlatButton
          backgroundColor="#a4c639"
          hoverColor="#8AA62F"
          icon={<IconSearch/>}
          onClick={this.handleClick}

        />
      </div>
    );
  }
}