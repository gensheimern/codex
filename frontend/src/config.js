const config = {
	debug: process.env.REACT_APP_DEBUG === 'TRUE',
	apiPath: process.env.REACT_APP_API_PATH,
	wsPath: process.env.REACT_APP_WS_PATH,
	basePath: process.env.REACT_APP_BASE_PATH,
}

export default config;