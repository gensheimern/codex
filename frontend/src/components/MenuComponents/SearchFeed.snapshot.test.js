import React from 'react';
import renderer from'react-test-renderer';
import SearchFeed from './SearchFeed';
import config from '../../config';
import {MemoryRouter} from 'react-router-dom';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
const fetchMock = require('fetch-mock');

describe('SearchFeed Snapshot', () => {
    test('renders', () =>{
        const component = renderer.create(
            <MemoryRouter>
                <MuiThemeProvider>
                    <SearchFeed/>
                </MuiThemeProvider>
            </MemoryRouter>
        );
        let tree= component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});