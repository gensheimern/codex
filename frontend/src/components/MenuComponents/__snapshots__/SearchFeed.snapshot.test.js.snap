// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchFeed Snapshot renders 1`] = `
<button
  disabled={false}
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  onTouchStart={[Function]}
  style={
    Object {
      "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
      "backgroundColor": "#1EA185",
      "border": 10,
      "borderRadius": 2,
      "boxSizing": "border-box",
      "color": "rgba(0, 0, 0, 0.87)",
      "cursor": "pointer",
      "display": "inline-block",
      "float": "left",
      "fontFamily": "Roboto, sans-serif",
      "fontSize": "inherit",
      "fontWeight": "inherit",
      "height": "100%",
      "lineHeight": "36px",
      "margin": 0,
      "minWidth": "56px",
      "muiPrepared": true,
      "outline": "none",
      "overflow": "hidden",
      "padding": 0,
      "position": "relative",
      "textAlign": "center",
      "textDecoration": "none",
      "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
      "userSelect": "none",
      "verticalAlign": null,
      "width": "56px",
    }
  }
  tabIndex={0}
  type="button"
>
  <div
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
  >
    <svg
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      style={
        Object {
          "color": "rgba(0, 0, 0, 0.87)",
          "display": "inline-block",
          "fill": "white",
          "height": 24,
          "marginLeft": 0,
          "marginRight": 0,
          "muiPrepared": true,
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "userSelect": "none",
          "verticalAlign": "middle",
          "width": 24,
        }
      }
      viewBox="0 0 24 24"
    >
      <path
        d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
      />
    </svg>
  </div>
</button>
`;
